attachment_log/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Attachment Log",
```

Comment:
Name is irrelevant because you are not adding any logging. There is no attachment log implemented.
It either logging or tracking.
Suggested name could be "Attachment Logging" with the technical name adjusted accordingly.